common/src/main/java/feast/common/logging/entry/AuditLogEntry.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright 2018-2019 The Feast Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package feast.common.logging.entry;
+
+import com.google.auto.value.AutoValue;
+import com.google.gson.Gson;
+
+/**
+ * AuditLogEntry represents a single audit Log Entry. Audit log entry can converted into string with
+ * {{@link #toString()} for human readable representation. Or structured JSON with {{@link
+ * #toJSON()} for a machine parsable representation.
+ */
+public abstract class AuditLogEntry {
+  /** Declare Log Type to allow external Logging systems to filter out {@link AuditLogEntry} */
+  public final String logType = "FeastAuditLogEntry";
+
+  /**
+   * FeastInstance provides metadata on the running instance which created the {@link AuditLogEntry}
+   */
+  @AutoValue
+  public abstract static class FeastInstance {
```

Comment:
I'm still hopeful that we can replace this class with just FeastProperties or one of its subclasses.

Reply:
See above comment about consolidating behind FeastProperties